### DISCUSSION TITLE
How to solve the problem： ModuleNotFoundError

### DISCUSSION THREAD

**Discussion by lala0228** [pytest-dev/pytest#12771] [Q&A]
_ ERROR collecting test/test_issue.py _______________________________________________
ImportError while importing test module '/home/lala/mini-testing-artifact-main/Tools/kuzu/tools/python_api/test/test_issue.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../.local/lib/python3.10/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
../../../.local/lib/python3.10/site-packages/_pytest/pathlib.py:582: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
    ???
<frozen importlib._bootstrap>:1027: in _find_and_load
    ???
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:688: in _load_unlocked
    ???
../../../.local/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:174: in exec_module
    exec(co, module.__dict__)
tools/python_api/test/test_issue.py:3: in <module>
    from tools.python_api.test.type_aliases import ConnDB
E   ModuleNotFoundError: No module named 'tools.python_api'

**Comment by dongfangtianyu** [comment]
Hi @lala0228  ,
It seems that the problem lies in this line of code
```python
# tools/python_api/test/test_issue.py:3:
from tools.python_api.test.type_aliases import ConnDB
```

Can you provide more detailed information that would help with the analysis of the problem？
For example, 
in which directory was pytest launched?
What command and parameters were used to start pytest?
Is there pytest.ini? If so, what is the content in it?
And the complete content after starting pytest.


You can replace the secret characters with `***`

**Comment by The-Compiler** [comment]
Converting this to a discussion as this is not a bug in pytest. Please use those rather than issues for support questions.
